parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -137,13 +137,15 @@ installMoby() {
 installKataContainersRuntime() {
     # TODO incorporate this into packer CI so that it is pre-baked into the VHD image
     echo "Adding Kata Containers repository key..."
+    ARCH=x86_64
+    BRANCH=stable-1.7
```

Comment:
where does stable-1.7 come from?

Reply:
From http://download.opensuse.org/repositories/home:/katacontainers:/releases:/x86_64:/
Is there a better place to track the version in use? I assumed pinning to master isn't ideal

Reply:
nope, just curious. I agree, let's not pin to master :)